### DISCUSSION TITLE
DTSTART property required for VJOURNAL component

### DISCUSSION THREAD

**Discussion by danielgrigore** [ical4j/ical4j#710] [General]
Hi @benfortuna 

[rfc5545#section-3.6.3](https://datatracker.ietf.org/doc/html/rfc5545#section-3.6.3) for VJOURNAL component specifies that only **dtstamp** and  **uid** properties are required.

However the validator for [VJOURNAL component](https://github.com/ical4j/ical4j/blob/a8c0d43840fb76b07fbfea4d279809567db71618/src/main/java/net/fortuna/ical4j/validate/ComponentValidator.java#L98) makes **DTSTART** property mandatory besides uid and dtstamp.

`...
public static final ComponentValidator<VJournal> VJOURNAL = new ComponentValidator<>(Component.VJOURNAL,
            new ValidationRule<>(One, DTSTART, DTSTAMP, UID),
....
`

Are there any reasons for this or is it maybe a bug ?
Thanks.

**Comment by benfortuna** [comment]
Apologies for delay. Yes most likely a bug, although I will check the other specs to see if this was modified by a later one. If not I will fix for the next release.

**Reply by duncan-blues** [reply]
Has this ever been addressed?
